system-variables.md

```diff
@@ -321,14 +322,14 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 
 - Scope: SESSION | GLOBAL
 - Default value: `10240`
-- Range: `[-1, 9223372036854775807]`
+- Range: `[0, 9223372036854775807]`
```

Comment:
This is actually wrong. In the source code you can see that it has a minValue of 0, and does not permit the special autovalue of -1:

https://github.com/pingcap/tidb/blob/22034c5bcd690dda3a9e794d6057cb5d9385d7cf/sessionctx/variable/sysvar.go#L876-L879

Reply:
This was introduced in https://github.com/pingcap/docs-cn/pull/6875/files#diff-ec2cbb115cc58fe3afe2862eb5148732d323cbf7ae35dea97bd381a1f8d0e8a0R331. @hanfei1991 @LittleFall Could you help confirm whether the minimum value should be 0 or -1? Thanks!

Reply:
```
5.7.25-TiDB-v5.2.0-alpha-720-gd9bf3bf2d 127.0.0.1:4000   test  SQL  set session tidb_broadcast_join_threshold_count=-1;
ERROR: 1231 (42000): Variable 'tidb_broadcast_join_threshold_count' can't be set to the value of '-1'
```

Reply:
Sorry to hear this. The reason for this error is that although we have implemented the corresponding logic, we have not expanded the scope of the variable, which results in behavior that is inconsistent with the document.
Thanks for pointing it out!
We will modify this document to be consistent with the behavior, and fix this error in 5.2.1.